### DISCUSSION TITLE
Raspberry Pi 3 + Ubuntu 24.04 LTS

### DISCUSSION THREAD

**Discussion by SvenVD** [OpenSurv/OpenSurv#20] [Hardware + Operating system]
All experiences with Raspberry Pi 3 + Ubuntu 24.04 LTS

**Comment by UntouchedWagons** [comment]
Where'd you get 24.04 for the Pi 3? The latest Ubuntu I can find for the Pi 3 is 22.04.

**Comment by SvenVD** [comment]
You are correct, I never tested this combination myself. I will close this thread
